swagger.yaml

```diff
@@ -627,6 +627,87 @@ paths:
           schema:
             $ref: '#/definitions/error'
 
+  /clusters/{cluster_id}/discovery-ignition:
```

Comment:
@avishayt do we need a `security` part in the APIs?

Reply:
We need it only if the default is not appropriate.
The default is UserAuth.
In this case I don't think we need to override

Reply:
Thanks